chia/protocols/harvester_protocol.py

```diff
@@ -71,6 +91,7 @@ class RespondSignatures(Streamable):
     local_pk: G1Element
     farmer_pk: G1Element
     message_signatures: List[Tuple[bytes32, G2Element]]
+    farmer_reward_address_overwrite: Optional[bytes32]
```

Comment:
not important; my model is that this is a "diverge" rather than an "overwrite". The latter suggests that we first put the default farmer reward address in, and then overwrite it. Perhaps we implement it like that, but it's not an important distinction.

Reply:
That's precisely why I changed it to "overwrite" (originally named it override), since we are indeed overwriting the default one. I've no problem going back to override, as it seems more intuitive